onnx/__init__.py

```diff
@@ -121,7 +135,7 @@ def _serialize(proto: Union[bytes, google.protobuf.message.Message]) -> bytes:
 _Proto = TypeVar("_Proto", bound=google.protobuf.message.Message)
 
 
-def _deserialize(s: bytes, proto: _Proto) -> _Proto:
+def _deserialize(s: bytes, proto: _Proto, format_: _SupportedFormat) -> _Proto:
```

Comment:
I would keep `Union[Literal["protobuf"], Literal["textproto"]]`. It is easier to understand for users or it should be documented and not added as a private attribute of the module.

Reply:
I think we should see it expanded in the sphinx docs? I tested in vscode and intellisense was very good at telling me what are the allowed values. I put them in a single type alias considering it will be easier when we expand this set?